src/components/Header/NavBar.js

```diff
@@ -2,36 +2,45 @@ import Link from 'next/link';
 import Head from 'next/head';
 import Image from 'next/image';
 import styles from './NavBar.module.css';
+import ReusableNavLink from '../common/ReusableNavLink';
+
 
 export default function NavBar() {
+    const navList = [
+        { text: "About", href: "/about" },
```

Comment:
@istoga Do we actually have these pages? 👀  If not, do we want them? Perhaps a topic to raise in Slack.

Reply:
Great point, I will ask via Slack.

Reply:
Seems overall consent is we should nix the pages.